aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3MultipartUpload.java

```diff
@@ -63,7 +64,13 @@ public static void beforeClass() {
     properties = new S3FileIOProperties();
     properties.setMultiPartSize(S3FileIOProperties.MULTIPART_SIZE_MIN);
     properties.setChecksumEnabled(true);
-    io = new S3FileIO(() -> s3, properties);
+    io = new S3FileIO(() -> s3);
+    io.initialize(
+        ImmutableMap.of(
+            S3FileIOProperties.MULTIPART_SIZE,
+            Integer.toString(S3FileIOProperties.MULTIPART_SIZE_MIN),
+            S3FileIOProperties.CHECKSUM_ENABLED,
+            "true"));
```

Comment:
Nitpicking: Maybe Boolean.TRUE.toString() is a bit more descriptive and static than a String literal.

Reply:
Since this is for tests there's really no benefit in using `Boolean.TRUE.toString()` vs the plain string

Reply:
Yeah I actually think the explicit "true" is better in this case tbh